doc/source/snippets/doc-local-build.rst

```diff
@@ -1,25 +1,61 @@
-#. Install the requirements for building the documentation.
+#. Install documentation related dependencies:
 
     .. code-block:: bash
 
         $ conda install --file requirements/docs.txt
 
-#. Then we will use an external tool called ``sphinx-reload`` to automatically reload the documentation when you make changes to ``.rst`` files.
+#. Enter into the ``doc`` project directory and render documentation:
```

Comment:
As discussed, not using `sphinx-reload`. But using the way that works for everyone.